src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -1189,23 +1183,19 @@ private static void verifyHeader(byte[] data) {
         checkRenderingIntent(data);
     }
 
-    private static boolean checkRenderingIntent(byte[] header) {
+    private static void checkRenderingIntent(byte[] header) {
         int index = ICC_Profile.icHdrRenderingIntent;
-
-        /* According to ICC spec, only the least-significant 16 bits shall be
-         * used to encode the rendering intent. The most significant 16 bits
-         * shall be set to zero. Thus, we are ignoring two most significant
-         * bytes here. Please refer ICC Spec Document for more details.
+        /*
+         * ICC spec: only the least-significant 16 bits encode the rendering
+         * intent. The most significant 16 bits must be zero and can be ignored.
+         * See https://www.color.org/ICC1v42_2006-05.pdf, section 7.2.15.
```

Comment:
ICC Spec link in the comment refers to older one, probably update to the new version ?  https://www.color.org/specification/ICC.1-2022-05.pdf , https://www.color.org/icc_specs2.xalter.